cluster/manager.go

```diff
@@ -1394,6 +1433,15 @@ func (c *ClusterManager) NodeRemoveDone(nodeID string, result error) {
 
 	err := c.deleteNodeFromDB(nodeID)
 	if err != nil {
+		goto error_done
+	}
+	_, err = c.discoverService.RemoveNode(discovery.NodeEntry{Id: nodeID})
```

Comment:
what if we crash before removing it from discoverService but it is removed from all other dbs ?
(This is a third db that needs to be kept in sync with the /storage and /cluster ).